salt/metalk8s/roles/etcd/init.sls

```diff
@@ -1,3 +1,3 @@
 include:
-  - metalk8s.roles.internal.node-without-calico
-  - metalk8s.kubernetes.etcd
+  - metalk8s.kubernetes.etcd.configured
```

Comment:
This state does not exist and IMO you should not change the roles state but only include the `prepared` in the `metalk8s.kubernetes.etcd.init`or directly in `metalk8s.kubernetes.etcd.installed`